### PR TITLE
Release/0.0.4rc1

### PR DESCRIPTION
**Pull request by mixmix**
- CHANGELOG complete
- small fixes
  - README update
  - CLI command syntax fix (spotted when calling `--help`)
  - test rename (consistency)
- deployed as `0.0.4-1`
  - `npm publish --tag next` works a charm!

## Try it out

```bash
npm i -g @entropyxyz/cli@0.0.4-1
```

We'll need to wait for test net to be up ... 

<details>
  <summary>
    <b>OR  run your own >>> </b>
  </summary>
```
git clone git@github.com:entropyxyz/sdk.git
cd sdk
./dev/bin/spin-up.sh four-nodes
docker ps
```

Set the endpoint
```
export ENTROPY_ENDPOINT=dev
```

To spin down 
```bash
./dev/bin/spin-up.sh four-nodes
````
</details>

Away you go!

```bash
entropy --help
entropy account create mixy
```

### CODE REVIEW

**Comment by mixmix** [comment]
Early thing I notice is it takes 5seconds to start the TUI :melting_face:

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,78 +13,121 @@ Version header format: `[version] Name - year-month-day (entropy-core compatibil
...
 
 ### Broke
 
-- deploying programs with TUI
-    - now requires a `*.wasm` file for `bytecode`
-    - now requires a `*.json` file path for `configurationSchema`
-    - now requires a `*.json` file path for `auxillaryDataSchema`
-
+- network now uses `four-nodes` docker setup
+  - requires an update to `/etc/hosts` for local testsing, should include line:
```
Comment (rh0delta) [inline CHANGELOG.md:79]:
```suggestion
  - requires an update to `/etc/hosts` for local testing, should include line:
```

**Review by rh0delta — APPROVED** [review]
Gonna approve, just a minor spelling mistake

**Comment by ameba23** [comment]
I had a go at using this with the docker compose setup (using the entropy-core docker compose script, not the SDK docker compose script), and was not able to register. 

I have the feeling it is because of something mistake on my side rather than actually being a code issue, but i made an issue anyway: https://github.com/entropyxyz/cli/issues/268

**Comment by mixmix** [comment]
There might be gotchas @ameba23 
- you need to jump start your network? (I think can to via tui only)
- you need a funded account to register, I think you need a funded seed to transfer or to use the faucet (not sure if that's deployed by default?)

**Comment by mixmix** [comment]
We are going to cut another release candidate, rc2

Merging this into `dev` for now

**Pull request merged**